docs/ssr.mdx

```diff
@@ -39,10 +39,8 @@ You can also use the advanced integration, it requires more work but does not ha
 import { CacheProvider } from '@emotion/react'
 import { renderToString } from 'react-dom/server'
 import createEmotionServer from '@emotion/server/create-instance'
-import createCache from '@emotion/cache'
+import { cache } from '@emotion/css'
```

Comment:
This is actually not because what is documented doesn't work but rather because you are using `@emotion/css` rather than `@emotion/react`.

This would actually need a separate recipe below this one to document your solution that is intended to work with `@emotion/css`. In your case most likely there is also no need to use `@emotion/server/create-instance` - `import { extractCritical } from '@emotion/server'` probably works just fine.

Reply:
ok got it, that makes sense. I got it mixed up. 

Let me add a new section under `@emotion/css`. I got an example running in [codesandbox](https://codesandbox.io/s/festive-water-b374y?file=/src/index.ts), if it looks good, i'll add it.

Reply:
Yes, it looks OK. Please try to add the section where you see fit and we'll continue discussing from there.